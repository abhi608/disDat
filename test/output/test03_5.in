Starting T1
Starting T2
T1 got read lock on x3 having value 30
T2 got write lock on x8
Site: 2 failed
T2 aborted as site 2 failed
DEBUG| Clearing site locks for T2 variable: x8
Clearing locks for T2 variable: x8
T1 got write lock on x5
Site: 2 recovered
T1 read the value 30 of variable x3
T1 committed
DEBUG| Clearing site locks for T1 variable: x3
Clearing locks for T1 variable: x3
DEBUG| Clearing site locks for T1 variable: x5
Clearing locks for T1 variable: x5
=== Site 1 ===
All other variables have same initial value
=== Site 2 ===
x20 is not available for reading
x18 is not available for reading
x14 is not available for reading
x16 is not available for reading
x12 is not available for reading
x11: 110 (available at site 2 for reading as it is the only copy or has been written after recovery)
x10 is not available for reading
x4 is not available for reading
x1: 10 (available at site 2 for reading as it is the only copy or has been written after recovery)
x2 is not available for reading
x8 is not available for reading
x6 is not available for reading
=== Site 3 ===
All other variables have same initial value
=== Site 4 ===
All other variables have same initial value
=== Site 5 ===
All other variables have same initial value
=== Site 6 ===
x5: 91 at site 6
All other variables have same initial value
=== Site 7 ===
All other variables have same initial value
=== Site 8 ===
All other variables have same initial value
=== Site 9 ===
All other variables have same initial value
=== Site 10 ===
All other variables have same initial value
