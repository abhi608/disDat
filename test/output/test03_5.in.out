Starting T1
Starting T2
T1 got read lock on x3 having value 30
T2 got write lock on x8
Site: 2 failed
T2 aborted as site 2 failed
DEBUG| Clearing site locks for T2 variable: x8
Clearing locks for T2 variable: x8
T1 got write lock on x5
Site: 2 recovered
T1 read the value 30 of variable x3
T1 write the value 91 of variable x5
T1 committed
DEBUG| Clearing site locks for T1 variable: x3
Clearing locks for T1 variable: x3
DEBUG| Clearing site locks for T1 variable: x5
Clearing locks for T1 variable: x5
=== Site 1 ===
x20: 200 at site 1
x16: 160 at site 1
x14: 140 at site 1
x10: 100 at site 1
x8: 80 at site 1
x6: 60 at site 1
x4: 40 at site 1
x18: 180 at site 1
x12: 120 at site 1
x2: 20 at site 1
=== Site 2 ===
x20 is not available for reading
x16 is not available for reading
x14 is not available for reading
x8 is not available for reading
x6 is not available for reading
x11: 110 (available at site 2 for reading as it is the only copy or has been written after recovery)
x4 is not available for reading
x18 is not available for reading
x12 is not available for reading
x2 is not available for reading
x10 is not available for reading
x1: 10 (available at site 2 for reading as it is the only copy or has been written after recovery)
=== Site 3 ===
x20: 200 at site 3
x16: 160 at site 3
x14: 140 at site 3
x10: 100 at site 3
x8: 80 at site 3
x6: 60 at site 3
x4: 40 at site 3
x18: 180 at site 3
x12: 120 at site 3
x2: 20 at site 3
=== Site 4 ===
x20: 200 at site 4
x16: 160 at site 4
x14: 140 at site 4
x10: 100 at site 4
x6: 60 at site 4
x4: 40 at site 4
x13: 130 at site 4
x8: 80 at site 4
x3: 30 at site 4
x18: 180 at site 4
x12: 120 at site 4
x2: 20 at site 4
=== Site 5 ===
x20: 200 at site 5
x16: 160 at site 5
x14: 140 at site 5
x10: 100 at site 5
x8: 80 at site 5
x6: 60 at site 5
x4: 40 at site 5
x18: 180 at site 5
x12: 120 at site 5
x2: 20 at site 5
=== Site 6 ===
x20: 200 at site 6
x14: 140 at site 6
x15: 150 at site 6
x10: 100 at site 6
x8: 80 at site 6
x6: 60 at site 6
x16: 160 at site 6
x5: 91 at site 6
x4: 40 at site 6
x18: 180 at site 6
x12: 120 at site 6
x2: 20 at site 6
=== Site 7 ===
x20: 200 at site 7
x16: 160 at site 7
x14: 140 at site 7
x10: 100 at site 7
x8: 80 at site 7
x6: 60 at site 7
x4: 40 at site 7
x18: 180 at site 7
x12: 120 at site 7
x2: 20 at site 7
=== Site 8 ===
x17: 170 at site 8
x16: 160 at site 8
x14: 140 at site 8
x10: 100 at site 8
x8: 80 at site 8
x20: 200 at site 8
x7: 70 at site 8
x6: 60 at site 8
x4: 40 at site 8
x18: 180 at site 8
x12: 120 at site 8
x2: 20 at site 8
=== Site 9 ===
x20: 200 at site 9
x16: 160 at site 9
x14: 140 at site 9
x10: 100 at site 9
x8: 80 at site 9
x6: 60 at site 9
x4: 40 at site 9
x18: 180 at site 9
x12: 120 at site 9
x2: 20 at site 9
=== Site 10 ===
x16: 160 at site 10
x19: 190 at site 10
x14: 140 at site 10
x10: 100 at site 10
x20: 200 at site 10
x9: 90 at site 10
x8: 80 at site 10
x6: 60 at site 10
x4: 40 at site 10
x18: 180 at site 10
x12: 120 at site 10
x2: 20 at site 10
